src/commands/explain.rs

```diff
@@ -169,14 +192,14 @@ pub fn explain(args: Args) -> anyhow::Result<()> {
                     }
                     None => {
                         println!("{rules}\n", rules = list_all_rules());
-                        bail!("No rule found with the name `{rule_name}`");
+                        bail!("no rule found with the name `{rule_name}`");
                     }
                 }
             }
         }
 
         Ok(())
     } else {
-        bail!("Invalid arguments: either a rule_name, a --tag, or --definitions must be provided");
+        bail!("invalid arguments: either a rule_name, a --tag, or --definitions must be provided");
```

Comment:
I think this will `unreachable!()`? If not, the message needs to be updated to include `list_all_[rules,tags]`